src/solver/libModelObject/include/antares/solver/libObjectModel/expression.h

```diff
@@ -28,8 +28,20 @@ namespace Antares::Solver::ObjectModel
 class Expression
 {
 public:
-    Expression();
-    ~Expression() = default;
+    Expression() = default;
+
+    explicit Expression(std::string value):
+        value(value)
```

Comment:
```suggestion
        value(std::move(value))
```

Reply:
since is not const&

Reply:
Since it's just an empty constructor and a getter I left them in header. No other reason